dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -1083,6 +1082,41 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     assert_equal csp.id, JSON.parse(@response.body).first['id']
   end
 
+  test 'workshop admins can unstart' do
+    @workshop.sessions << create(:pd_session)
+    @workshop.start!
+
+    sign_in @workshop_admin
+    post :unstart, params: {id: @workshop.id}
+    assert_response :success
+    @workshop.reload
+    assert_equal 'Not Started', @workshop.state
+  end
+
+  test 'workshop admins can reopen' do
+    @workshop.sessions << create(:pd_session)
+    @workshop.start!
+    @workshop.end!
+
+    sign_in @workshop_admin
+    post :reopen, params: {id: @workshop.id}
+    assert_response :success
+    @workshop.reload
+    assert_equal 'In Progress', @workshop.state
+  end
+
+  test 'workshop organizers cannot unstart' do
```

Comment:
We're trying to phase out workshop organizers. Suggest making this test for regional partner program managers